### ISSUE TITLE
Cannot save wiki page

### ISSUE THREAD

**Issue by michellealva** [frappe/wiki#174]
```
Traceback (most recent call last):
  File "apps/frappe/frappe/app.py", line 80, in application
    response = frappe.handler.handle()
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/handler.py", line 48, in handle
    data = execute_cmd(cmd)
           ^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/handler.py", line 86, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/__init__.py", line 1652, in call
    return fn(*args, **newargs)
           ^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/utils/typing_validations.py", line 30, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "apps/wiki/wiki/wiki/doctype/wiki_page/wiki_page.py", line 495, in update
    patch.submit()
  File "apps/frappe/frappe/model/document.py", line 1027, in submit
    return self._submit()
           ^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 1006, in _submit
    return self.save()
           ^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 326, in save
    return self._save(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 378, in _save
    self.run_post_save_methods()
  File "apps/frappe/frappe/model/document.py", line 1111, in run_post_save_methods
    self.run_method("on_submit")
  File "apps/frappe/frappe/model/document.py", line 936, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 1302, in composer
    return composed(self, method, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 1284, in runner
    add_to_return_value(self, fn(self, *args, **kwargs))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 933, in fn
    return method_object(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/wiki/wiki/wiki/doctype/wiki_page_patch/wiki_page_patch.py", line 35, in on_submit
    self.create_new_wiki_page()
  File "apps/wiki/wiki/wiki/doctype/wiki_page_patch/wiki_page_patch.py", line 57, in create_new_wiki_page
    self.new_wiki_page.save()
  File "apps/frappe/frappe/model/document.py", line 326, in save
    return self._save(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 348, in _save
    return self.insert()
           ^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 279, in insert
    self.run_before_save_methods()
  File "apps/frappe/frappe/model/document.py", line 1072, in run_before_save_methods
    self.run_method("before_save")
  File "apps/frappe/frappe/model/document.py", line 936, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 1302, in composer
    return composed(self, method, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 1284, in runner
    add_to_return_value(self, fn(self, *args, **kwargs))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 933, in fn
    return method_object(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/wiki/wiki/wiki/doctype/wiki_page/wiki_page.py", line 29, in before_save
    self.content = self.sanitize_html()
                   ^^^^^^^^^^^^^^^^^^^^
  File "apps/wiki/wiki/wiki/doctype/wiki_page/wiki_page.py", line 119, in sanitize_html
    escaped_html = bleach.clean(
                   ^^^^^^^^^^^^^
TypeError: clean() got an unexpected keyword argument 'styles'
```

**Comment by ankush** [comment]
refer https://github.com/frappe/frappe/pull/21782

**Comment by BreadGenie** [comment]
fixed by 23689b403e0b56b622fb36843812e2b2bd937303
